.env.sample

```diff
@@ -133,3 +133,5 @@ REACT_APP_PLAN_TYPES_WITH_MULTI_JURISDICTIONS='IRS,Dynamic-IRS,MDA-Point,Dynamic
 
 # options displayed when downloading jurisdiction metadata
 REACT_APP_ENABLED_JURISDICTION_METADATA_IDENTIFIER_OPTIONS=POPULATION,RISK,COVERAGE,TARGET
+# max number of plans in an openSRP service call
+REACT_APP_OPENSRP_MAX_PLANS_PER_REQUEST = 2000
```

Comment:
There's a [doc for envs](https://github.com/onaio/reveal-frontend/blob/master/docs/env.md) where this should be added to.

Reply:
ooh yeha